otx/algorithms/detection/adapters/mmdet/nncf/compression.py

```diff
@@ -255,12 +220,14 @@ def dummy_forward(model):
             if is_alt_ssd_export:
                 img = img[0]
                 img_metas = img_metas[0]
-            ctx = model.forward_export_context(img_metas)
+            ctx = model.nncf_trace_context(img_metas)
             logger.debug(f"NNCF will compress a postprocessing part of the model")
         else:
             ctx = model.forward_dummy_context(img_metas)
             logger.debug(f"NNCF will NOT compress a postprocessing part of the model")
         with ctx:
+            # The device where model is could be changed under this context
+            img = [i.to(next(model.parameters()).device) for i in img]
```

Comment:
forward_dummy_context() also should be moved or fetched to somewhere else.

Reply:
This is resolved in a branch where I am working on `mmdeploy`. Let me cherry-pick relevant commits into this branch.